### PR TITLE
Fixed build for Homebrew 1.1.2 on Sierra 10.12.1

### PR DESCRIPTION
**Pull request by mvalkema**

### CODE REVIEW

File: packaging/homebrew/cassandra-cpp-driver.rb [hunk 1]
```diff
@@ -3,17 +3,19 @@
...
 
   depends_on "cmake" => :build
   depends_on "libuv"
+  depends_on "openssl"
+  depends_on "boost160"
 
   def install
     mkdir 'build' do
-      system "cmake", "-DCMAKE_BUILD_TYPE=RELEASE", "-DCASS_BUILD_STATIC=ON", "-DCASS_INSTALL_PKG_CONFIG=OFF", "-DCMAKE_INSTALL_PREFIX:PATH=#{prefix}", "-DCMAKE_INSTALL_LIBDIR=#{lib}", ".."
+      system "cmake", "-DCMAKE_BUILD_TYPE=RELEASE", "-DCASS_BUILD_STATIC=ON", "-DCASS_INSTALL_PKG_CONFIG=OFF", "-DCMAKE_INSTALL_PREFIX:PATH=#{prefix}", "-DCMAKE_INSTALL_LIBDIR=#{lib}",".."
```
Comment (stamhankar999) [inline packaging/homebrew/cassandra-cpp-driver.rb]: This seems to be just a whitespace change, and one for the worse. Can you revert this line?

Comment (mvalkema) [inline packaging/homebrew/cassandra-cpp-driver.rb]: My bet. That was a leftover from some playing around with the file. I reverted the line, and commited the change. Feel free to squash the 2 commits.

File: packaging/homebrew/cassandra-cpp-driver.rb [hunk 2]
```diff
@@ -3,13 +3,15 @@
...
-  sha1 "6159f1a5e31a044fb70567f849286572afa57174"
+  sha256 "5a7325057231b65ad36359fe07d37b0dfbf2a522fa6961138c5efbaa7b265831"
   version "2.2.0-beta1"
 
   head "git://github.com:datastax/cpp-driver.git", :branch => "master"
 
   depends_on "cmake" => :build
   depends_on "libuv"
+  depends_on "openssl"
+  depends_on "boost160"
```
Comment (mpenick) [inline packaging/homebrew/cassandra-cpp-driver.rb]: Why was boost added as a dependency? Only tests require boost and the formula doesn't build the tests.

**Comment by mikefero** [comment]
Thanks for the contribution, resolved by #358

**Pull request closed without merging**